README.md

```diff
@@ -11,49 +11,84 @@
   </a>
 </p>
 
-This build plugin is a utility for enabling server-side rendering in Next.js on Netlify. It wraps your application in a tiny compatibility layer, so that pages can use Netlify Functions to be server-side rendered.
+This build plugin is a utility for supporting Next.js on Netlify. To enable server-side rendering and other framework-specific features in your Next.js application on Netlify, your app will need to install this plugin.
 
 ## Table of Contents
 
 - [Installation and Configuration](#installation-and-configuration)
-- [Custom Netlify Redirects](#custom-netlify-redirects)
 - [Custom Netlify Functions](#custom-netlify-functions)
+- [Publish Directory](#publish-directory)
+- [Custom Netlify Redirects](#custom-netlify-redirects)
 - [Caveats](#caveats)
-  - [Fallbacks for Pages with `getStaticPaths`](#fallbacks-for-pages-with-getstaticpaths)
 - [Credits](#credits)
 - [Showcase](#showcase)
 
 ## Installation and Configuration
 
-Create a `netlify.toml` in the root of your project:
+Create a `netlify.toml` in the root of your project. Your .toml should include:
+
+```toml
+[build]
+  command   = "npm run build"
+
+[[plugins]]
+  package = "@netlify/plugin-nextjs"
+```
+
+This plugin can also be installed and managed from your site's settings on Netlify.
+If you want to install there, you can omit this [[plugins]] section from your .toml file.
+
+## Custom Netlify Functions
```

Comment:
Do we even need to mention custom Netlify Functions at all?

I am assuming if a site is using a Functions directory, they will assume this plugin to just work with it.
If a site is not using a Functions directory, then the default Functions directory will be used instead, which the user will not see.

Reply:
tabling for after <3